.circleci/config.yml

```diff
@@ -39,7 +39,7 @@ jobs:
       - checkout
       - run:
           name: ZIP the Codebase
-          command: zip logdna-go.zip -r logger/logdna.go go.mod
+          command: zip logdna-go.zip -r logger.go logger_test.go options.go options_test.go go.mod
```

Comment:
Cross compiling Go is rather trivial these days. Rather that releasing a zip of source, why not provide pre-built binaries?
Or better, provide both. Source zip and prebuilt binaries for win, macos + linux.  You don't even need the native OS to do it in most cases

```shell
# build for macos on linux
GOOS=darwin GOARCH=amd64  go build logger.go
```

```shell
# build for windows on linux
GOOS=windows and GOARCH=386 go build logger.go
```

Thats a pretty simple first step before with out needing to figure out OS package managers.
Should probably also provide a `tar.gz` file which is more common place on *nix systems.

Make it easy for people. Most users would rather not have to compile things.

Reply:
@mdeltito i think that was a change implemented by @smusali so will delegate to him to answer that question- but yup looks like the circleci config zips and packages it for release- i assume this is similar to what we do in other logdna projects